Tools/Scripts/webkitdirs.pm

```diff
@@ -2448,7 +2448,7 @@ sub shouldUseFlatpak()
         return 0;
     }
 
-    if (defined $ENV{'WEBKIT_JHBUILD'} && $ENV{'WEBKIT_JHBUILD'}) {
+    if ((defined $ENV{'WEBKIT_JHBUILD'} && $ENV{'WEBKIT_JHBUILD'}) or defined $ENV{'WKDEV_SDK'}) {
```

Comment:
Well you should check its value just like with `WEBKIT_JHBUILD` or else there's no difference between `WKDEV_SDK=1` vs. `WKDEV_SDK=0`

Reply:
This value is set to a string, being anything should be good enough. It isn't like you accidentally set it.

Reply:
Oh OK